modules/ROOT/pages/managing-business-groups.adoc

```diff
@@ -16,17 +19,150 @@ What you can view and edit depends on your permissions.
+--
+
+The business group hierarchy prevents a child business group from consuming additional resources from its parent group unless an organization administrator redistributes those resources to the child group. Resource distribution proceeds from root to parent to child. For example, you can't redistribute sandbox VCores to a child business group if the parent business group doesn't have any sandbox vCores available, even if the root organization has sandbox vCores available.
+
+Additionally, the root organization or parent business group can't reclaim resources allocated to a business group that isn't a direct child.
+
+. In the Access Management navigation menu, click *Business Groups*.
+. Click the name of the business group to reassign resources to.
++
+The *Settings* section appears, showing details about the root organization or business group.
+. Click the *Settings* tab.
+. Enter the number of vCores you want the business group to have.
+. Click *Save changes*.
+
+The *Business Group Info* window includes a resource counter that shows a value representing available resources, the sum of those resources that are reassigned to child business groups, and resources currently in use by the selected business group.
+
+=== View Resource Utilization
+
+To see resource utilization for the root organization, navigate to *Access Management > Subscription > Runtime Manage Subscription*.
 
 [[client-id-and-secret]]
 == Client ID and Client Secret
 
-Each root organization, environment, and business group within the root organization has its own associated unique client ID and client secret. These are used for authentication by users who are not organization administrators to access assets within an organization. The client ID and password are generated by Anypoint Platform for each environment you create, and they are globally unique.
+Each root business group, child business group, and environment, has its own associated unique client ID and client secret. These are used for authentication by users who are not business group administrators to access assets within a business group. The client ID and password are generated by Anypoint Platform for each environment you create, and they are globally unique.
 
 To deploy proxies or APIs to CloudHub, you must use these values to configure a customer-hosted Mule Runtime or legacy API Gateway.
 
 [NOTE]
-Organization-level client IDs and client secrets are supported only for backward compatibility. In newer Anypoint Platform accounts, use the client ID and client secret for an environment instead. See xref:environments.adoc[Environments].
+Business group-level client IDs and client secrets are supported only for backward compatibility. In newer Anypoint Platform accounts, use the client ID and client secret for an environment instead. See xref:environments.adoc[Environments].
```

Comment:
```suggestion
Business group-level client IDs and client secrets are supported only for backward compatibility. In newer Anypoint Platform accounts, use the client ID and client secret for an environment instead. See xref:environments.adoc[].
```

Reply:
Are client id and secret being deprecated in favor of connected apps?

Reply:
Typo on Line 177: Businesssssss

Reply:
@hannanelson Not as far as I know.